app/build.gradle

```diff
@@ -75,13 +75,15 @@ dependencies {
     implementation 'androidx.preference:preference-ktx:1.2.0'
 
     implementation 'com.github.cyb3rko:QuickPermissions-Kotlin:1.1.2'
-    implementation 'com.hypertrack:hyperlog:0.0.10'
     implementation 'com.squareup.picasso:picasso:2.71828'
     implementation 'io.noties.markwon:core:4.6.2'
     implementation 'io.noties.markwon:image-picasso:4.6.2'
     implementation 'io.noties.markwon:image:4.6.2'
     implementation 'io.noties.markwon:ext-tables:4.6.2'
     implementation 'io.noties.markwon:ext-strikethrough:4.6.2'
+
+    implementation 'org.tinylog:tinylog-api-kotlin:2.6.2'
+    runtimeOnly 'org.tinylog:tinylog-impl:2.6.2'
```

Comment:
On first start I had some issues becase the library was not found in GotifyApplication.kt.
But I'm not sure if the "runtimeOnly" could have been the issue or if this was just everday IDE craziness.

But it worked again after running it once.

Reply:
I'll change the tinylog-impl dependency to implementation. It's the way documented in their docs. This problem is still very likely unrelated, I did have similar problems after some dependency updates in previous PRs but they disappeared after clearing the cache.

Reply:
Where have you found this?  
Here it says to use "implementation" for Kotlin using Gradle (Groovy).

But it doesn't really have much of an impact.

Reply:
I meant that the the impl-dependency should be defined via implementation instead of runtimeOnly according to the tinylog docs. I've originally got this from an older project, and there tinylog had probably other docs (:.